assets/css/styles.css

```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
   }
 }
 
+/*
+  ADD PROJECT POPUP CSS
+  - Add the Popup class (same as the html file and style it)
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+
+.popup-modal {
+  position: absolute;
+  top: 0%;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--image-placeholder-color);
+  z-index: 100;
+}
+
+.project-popup {
+  width: 100%;
+  position: absolute;
+  background: var(--image-placeholder-color1);
+  border-radius: 1%;
+  top: 0;
+  left: 0;
+  transform: translate(-250%, -250%) scale(0.1);
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+}
+
+.project-popup-desktop-img {
+  width: 100%;
+  height: 60%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+.project-popup-mobile-img {
+  width: 100% !important;
+  height: 40%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
```

Comment:
In order to maintain a cleaner code base, kindly remove all commented codes.

Reply:
Done @emmiiorji